pkgs/top-level/python-packages.nix

```diff
@@ -29,7 +29,14 @@ let
   isPyPy = python.executable == "pypy";
   isPy3k = strings.substring 0 1 python.pythonVersion == "3";
 
-  callPackage = pkgs.newScope self;
+  callPackage = pkgs.newScope (self // sharedVersions);
+
+  # Some Python packages are released together and share the same version.
+  # These versions are collected in the sharedVersions attribute set and are
+  # added to the callPackage scope so that packages can refer to these versions.
+  sharedVersions = {
```

Comment:
I like the idea, but as there currently is only one version I don't think we should have this. If we were to have this, I think this should go to top-level so we could also use it with say `mercurial` and `tortoisehg` and other cases there.

Reply:
Another example: `libgit2` and `pygit2`.